### PR TITLE
Install remote execution plugin

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

**Comment by ekohl** [comment]
Would you mind writing a (minimal) spec?

**Comment by iNecas** [comment]
Tested and works well ACK

File: spec/classes/foreman_plugin_remote_execution_spec.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
+
+describe 'foreman::plugin::remote_execution' do
+  let :facts do
+    on_supported_os['redhat-7-x86_64']
```
Comment (domcleal) [inline spec/classes/foreman_plugin_remote_execution_spec.rb]: Use .each normally, so it compiles for every supported OS.

Comment (stbenjam) [inline spec/classes/foreman_plugin_remote_execution_spec.rb]: That seems like overkill, `foreman::plugin` is already well spec'd.  Salt was accepted before without one.  Should I add one for it too?

You'll be basically running the same spec a bunch of times.

Comment (domcleal) [inline spec/classes/foreman_plugin_remote_execution_spec.rb]: Yeah, it shouldn't have been accepted, or should have been fixed, which is why we're now asking for them - lack of coverage has caused problems.  Feel free to add one, but it's not really relevant here.

**Comment by mmoll** [comment]
merged, thanks @stbenjam

**Pull request closed without merging**